bin/express-cli.js

```diff
@@ -116,52 +132,30 @@ inquirer
 
     function createApplication (name, directory) {
       // Package
-      const pkg = {
-        name: kebabCase(name),
-        version: '1.0.0',
-        private: true,
-        scripts: {
-          'start': 'nodemon',
-          'build': 'npm-run-all clean transpile',
-          'server': 'node ./dist/bin/www',
-          'dev': 'NODE_ENV=development npm-run-all build server',
-          'prod': 'NODE_ENV=production npm-run-all build server',
-          'transpile': 'babel ./server --out-dir dist --copy-files',
-          'clean': 'rimraf dist'
-        },
-        nodemonConfig: {
-          'exec': 'npm run dev',
-          'watch': [
-            'server/*',
-            'public/*'
-          ],
-          'ignore': [
-            '**/__tests__/**',
-            '*.test.js',
-            '*.spec.js'
-          ]
-        },
-        dependencies: {
-          'babel-plugin-inline-dotenv': '^1.5.0',
-          'debug': '~2.6.9',
-          'express': '~4.16.1'
-        },
-        devDependencies: {
-          '@babel/cli': '^7.8.4',
-          '@babel/core': '^7.9.0',
-          '@babel/node': '^7.8.7',
-          '@babel/preset-env': '^7.9.0',
-          'jest': '^25.2.7',
-          'npm-run-all': '^4.1.5',
-          'rimraf': '^3.0.2',
-          'nodemon': '^2.0.3'
-        }
+      const pkg = { ...codeSnippets.pkg }
+      pkg.name = kebabCase(name)
+      if (hasTs) {
```

Comment:
We probably should move this logic to a separate file. its getting cluttered.

Reply:
Once we get this implemented, we can do some more cleanup. I'm all for organizing and beautifying it